tests/page/page-event-load.spec.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { test as it, expect } from './pageTest';
+
+
+it('should fire once', async ({ page, server, browserName }) => {
+  it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/15086' });
+  it.fixme(browserName === 'firefox', 'Firefox sometimes double fires.');
+
+  let count = 0;
+  page.on('load', () => count++);
+  await page.goto(server.PREFIX + '/one-style.html');
+  await page.waitForTimeout(1000);
```

Comment:
What's an alternative here? Asserting it fired at least once is easy, but not sure of a great way to assert **exactly** since we can't see into the future!